content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,200 @@
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+This page introduces the ServiceAccount object in Kubernetes, providing
+information about how service accounts work, use cases, limitations,
+alternatives, and links to resources for additional guidance.
+
+<!-- body -->
+
+## What are service accounts?
+
+A service account is a type of non-human account that, in Kubernetes, provides
+an identity for the processes that run in Pods. Pods can use service accounts
+to make calls to the Kubernetes API server. Service accounts exist as
+ServiceAccount objects in the API server. Service accounts have the following
+properties:
+
+*  **Namespaced:** Each service account is bound to a Kubernetes
+   {{<glossary_tooltip text="namespace" term_id="namespace">}}. Every namespace
+   gets a `default` ServiceAccount upon creation.
+
+*  **Lightweight:** Service accounts exist in the cluster and use the structure
+   defined in the Kubernetes API. You can quickly create service accounts to
+   enable specific tasks.
+
+*  **Portable:** A configuration bundle for a complex containerized workload
+   might include service account definitions for the system's components. The
+   lightweight nature of service accounts and the namespaced identities make
+   the configurations portable.
+
+Service accounts are different from user accounts, which are authenticated
+human users in the cluster. User accounts don't exist in the Kubernetes API
+server; instead, the API server treats user identities as opaque data. You can
+authenticate as a user account using multiple methods.
+
+{{< table caption="Comparison between service accounts and users" >}}
+
+| Description | ServiceAccount | User or group |
+| --- | --- | --- |
+| Location | Kubernetes API (ServiceAccount object) | External |
+| Scope | Namespace | Cluster |
+| Access control | Kubernetes RBAC or other [authorization mechanisms](/docs/reference/access-authn-authz/authorization/#authorization-modules) | Kubernetes RBAC or other identity and access management mechanisms |
+| Intended use | Pods | People |
+
+{{< /table >}}
+
+### Default service accounts
+
+When you create a cluster, Kubernetes automatically creates a ServiceAccount
+object named `default` for every namespace in your cluster. The `default`
+service accounts in each namespace get no permissions other than the
+[default API discovery permissions](/docs/reference/access-authn-authz/rbac/#service-account-permissions)
+that Kubernetes grants to all authenticated accounts. If you delete the default
+ServiceAccount object in a namespace, the {{<glossary_tooltip text="control plane" term_id="control-plane">}}
+replaces it with a new one.
+
+If you deploy a Pod in a namespace, and you don't manually assign a
+ServiceAccount object to the Pod, Kubernetes assigns the `default` service
+account for that namespace to the Pod. You can change this behavior by setting
+the `automountServiceAccountToken` field in your Pod specification to `false`.
```

Comment:
this isn't quite accurate... the `default` service account still gets assigned even when `autoMountServiceAccountToken` is false, but no token is auto-mounted.

Reply:
Done

Reply:
Actually done, idk where these changes disappeared to